apps/gauzy/src/app/pages/contacts/contact-mutation/contact-mutation.component.html

```
@@ -15,215 +15,212 @@ <h4>
-									class="green mr-2"
-									status="basic"
-									outline
-								>
-									{{ 'BUTTONS.NEXT' | translate }}
-								</button>
-								<button nbButton (click)="cancel()" status="basic" outline>
-									{{ 'BUTTONS.CANCEL' | translate }}
-								</button>
-							</div>
+						<div class="wizard-actions">
+							<button nbButton outline size="small" status="basic" class="wizard-quit" (click)="cancel()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)